presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -14,22 +14,55 @@
 package com.facebook.presto.cli;
 
 import com.facebook.presto.client.ClientSession;
+import com.facebook.presto.client.ErrorLocation;
+import com.facebook.presto.client.QueryError;
+import com.facebook.presto.client.QueryResults;
+import com.facebook.presto.client.StatementClient;
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import org.fusesource.jansi.Ansi;
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
+import java.util.List;
+
+import static com.facebook.presto.cli.ConsolePrinter.REAL_TERMINAL;
+import static com.google.common.base.Preconditions.checkState;
+import static java.lang.String.format;
 
 public class ErrorMessages
 {
+    private static final String TECHNICAL_DETAILS_HEADER = "\n=========   TECHNICAL DETAILS   =========\n";
+    private static final String SESSION_INTRO = "[ Session information ]\n";
+    private static final String ERROR_MESSAGE_INTRO = "[ Error message ]\n";
+    private static final String STACKTRACE_INTRO = "[ Stack trace ]\n";
+    private static final String TECHNICAL_DETAILS_END = "========= TECHNICAL DETAILS END =========\n\n";
+
     private ErrorMessages()
     {}
 
-    public static String createErrorMessage(Throwable throwable, ClientSession session)
+    public static String createQueryErrorMessage(StatementClient client)
+    {
+        return standardQueryErrorMessage(client);
+    }
+
+    public static String createExceptionMessage(Throwable throwable, ClientSession session)
```

Comment:
I wanted to make the difference in what does this method do in comparison to createQueryErrorMessage is clear.